cst/plugins/eu.esdihumboldt.cst.functions.string/plugin.xml

```diff
@@ -50,8 +50,8 @@
             </property>
          </targetProperties>
          <functionParameter
-               description="For example &quot;yyyy-MM-dd HH:mm:ss&quot;&#xA;(see http://docs.oracle.com/javase/6/docs/api/java/text/SimpleDateFormat.html)"
-               label="Date format"
+               description="For example: yyyy-MM-dd HH:mm:ss;&#xA; see http://docs.oracle.com/javase/6/docs/api/java/text/SimpleDateFormat.html for more details"
```

Comment:
I guess you might also want to remove the semicolon that belonged to the second `&quot;` then?